bitrise.yml

```diff
@@ -1248,6 +1244,26 @@ workflows:
         is_expand: false
       BITRISE_SCHEME: Firefox
     description: This step is to build, archive and upload Firefox Release and Beta
+  
+  SPM_Common_Steps:
```

Comment:
Grouping these steps that we run in all SPM_ workflows in one so that we don't have to repeat reading the version file step